interactive/templates/admin/registration_changeform.html

```diff
@@ -0,0 +1,8 @@
+{% extends 'admin/change_form.html' %}
+
+{% block submit_buttons_bottom %}
+    <div class="submit-row">
+            <input type="submit" value="Approve" name="_approve-request">
+            <input type="submit" value="Deny" name="_deny-request">
+    </div>
+{% endblock %}
```

Comment:
I've removed the save buttons, because I don't think a registration request should be changed, other than to approve/deny